cards.tex

```diff
@@ -46,6 +46,22 @@
 \newcommand{\Reference}[2]{{\small \color{gray} \engschrift #1 #2}}
 \newcommand{\DamageType}[1]{\operatorname{\mbox{#1}}}
 
+\newcommand{\Tag}[2]{%
+    \tikz[baseline]{%
+        \node[anchor=base, text=white, fill=#1, font=\sffamily, text depth=.5mm] {#2};
+    }%
+}
+
+\definecolor{tagsBg}{RGB}{217, 196, 132}
+\definecolor{uncommonBg}{RGB}{152, 81, 61}
+\definecolor{rareBg}{RGB}{0, 38, 100}
+\definecolor{traitBg}{RGB}{94, 0, 0}
+
+\newcommand{\Trait}[1]{\Tag{traitBg}{#1}}
+\newcommand{\Rare}[0]{\Tag{rareBg}{Rare}}
+\newcommand{\Uncommon}[0]{\Tag{uncommonBg}{Uncommon}}
```

Comment:
Is this okay if no arguments are required? Or would one do this differently?

Reply:
1. I think 0 is the default, so you can do `\newcommand{\Rare}{\Tag{rareBg}{Rare}}`
2. I think a nicer interface would be just `\tags{Rare, Uncommon, Magical, Extremely Cool, Overpowered}` and the `\tags` macro would know how to color which tag. That is a bit beyond my current LaTeX skills though :)